example/android/build.gradle

```diff
@@ -46,10 +47,23 @@ buildscript {
         getReactNativeDependencies().each { dependency ->
             classpath(dependency)
         }
+
+        if (isNewArchitectureEnabled(project)) {
+            classpath("com.facebook.react:react-native-gradle-plugin")
+            classpath("de.undercouch:gradle-download-task:5.1.0")
+        }
```

Comment:
This shouldn't be necessary. `getReactNativeDependencies()` should already have added these two: https://github.com/microsoft/react-native-test-app/blob/70f5d2b7bf6a2ded977f2d1cf164d79210927b69/android/dependencies.gradle#L104

Reply:
Removed